auslib/util/swagger.py

```diff
@@ -0,0 +1,119 @@
+import yaml
+import auslib
+
+from os import path
+
+AUS_DIR = path.dirname(auslib.__file__)
+
+
+class SpecBuilder():
+    def __init__(self,
+                 app,
+                 endpoints_key="paths",
+                 definitions_key="definitions",
+                 parameters_key="parameters",
```

Comment:
Are these really something you want consumers to be able to override? If I set the endpoints_key to "my_endpoints", would the spec still work? If they truly should be overridable, they belong as parameters to add_spec_part. If not, they should be class attributes.